src/DataMiner/SLNetTypes/Responses/GetProtocolInfoResponseMessage.cs

```diff
@@ -1,12 +1,1127 @@
+		/// </summary>
+		/// <param name="topologyname">The topology name.</param>
+		/// <param name="cell">The cell.</param>
+		/// <returns>The child tables.</returns>
+		public IEnumerable<ProtocolTopologyCell> FindChildTables(string topologyname, ProtocolTopologyCell cell)
+		{
+			if (cell == null)
+				yield break;
+		}
+
+		/// <summary>
+		/// Finds the child tables for the specified cell.
+		/// </summary>
+		/// <param name="topologyname">The topology name.</param>
+		/// <param name="tablePid">The table ID.</param>
+		/// <returns>The child tables.</returns>
+		public IEnumerable<ProtocolTopologyCell> FindChildTables(string topologyname, int tablePid)
+		{
+			if (Topologies == null)
+				yield break;
+		}
+
+		/// <summary>
+		/// Finds the parent tables for the specified cell.
+		/// </summary>
+		/// <param name="name">The name.</param>
+		/// <param name="cell">The cell.</param>
+		/// <returns>The parent tables.</returns>
+		public IEnumerable<ProtocolTopologyCell> FindParentTables(string name, ProtocolTopologyCell cell)
+		{
+			if (cell == null)
+				yield break;
+		}
+
+		/// <summary>
+		/// Finds the parent tables for the specified cell.
+		/// </summary>
+		/// <param name="topologyname">The topology name.</param>
+		/// <param name="tablePid">The table ID.</param>
+		/// <returns>The parent tables.</returns>
+		public IEnumerable<ProtocolTopologyCell> FindParentTables(string topologyname, int tablePid)
+		{
+			if (Topologies == null)
+				yield break;
+		}
+
+		/// <summary>
+		/// Returns all page names that are used by parameters in the protocol.
+		/// </summary>
+		/// <returns>All page names that are used by parameters in the protocol.</returns>
+		public string[] FindPageNames()
+		{
+			List<string> names = new List<string>();
+			return names.ToArray();
+		}
+
+		/// <summary>
+		/// Gets a value indicating whether the protocol has at least one parameter that can be used as a virtual source or destination (element connections).
+		/// </summary>
+		/// <value><c>true</c> if the protocol has at least one parameter that can be used as a virtual source or destination (element connections).</value>
+		public bool HasParametersForElementConnections
+		{
+			get;
+		}
+
+		/// <summary>
+		/// Returns information about which pages have been defined in the protocol, and which parameters are on each of these pages.
+		/// </summary>
+		/// <returns>Information about which pages have been defined in the protocol, and which parameters are on each of these pages.</returns>
+		/// <remarks>Only considers the actual device protocol. Does not return page info for the parameters of the base protocol.</remarks>
+		public PageInfo[] FindPages()
+		{
+			return null;
+		}
+
+		/// <summary>
+		/// Filters and/or sorts a collection of parameters, given a combination of selection options. e.g. returns a collection with only the analog parameters if <see pref="options"/> is set to <see cref="ParameterFilterOptions.AnalogOnly"/>.
```

Comment:
The `<see pref="options"/>` is not visible in my test build, leading to a weird sentence (same for line 975 below):

![image](https://github.com/SkylineCommunications/dataminer-docs/assets/94605575/f4c748cc-b7dc-4944-b183-63f890f4b517)

Reply:
corrected to <paramref name="options"/>